sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/implementation/TextAnalyticsClientImplBuilder.java

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.ai.textanalytics.implementation;
+
+import com.azure.core.annotation.ServiceClientBuilder;
+import com.azure.core.http.HttpPipeline;
+import com.azure.core.http.HttpPipelineBuilder;
+import com.azure.core.http.policy.CookiePolicy;
+import com.azure.core.http.policy.RetryPolicy;
+import com.azure.core.http.policy.UserAgentPolicy;
+
+/**
+ * A builder for creating a new instance of the TextAnalyticsClient type.
+ */
+@ServiceClientBuilder(serviceClients = TextAnalyticsClientImpl.class)
+public final class TextAnalyticsClientImplBuilder {
+    /*
+     * Supported Cognitive Services endpoints (protocol and hostname, for example: https://westus.api.cognitive.microsoft.com).
+     */
+    private String endpoint;
+
+    /**
+     * Sets Supported Cognitive Services endpoints (protocol and hostname, for example: https://westus.api.cognitive.microsoft.com).
+     *
+     * @param endpoint the endpoint value.
+     * @return the TextAnalyticsClientImplBuilder.
+     */
+    public TextAnalyticsClientImplBuilder endpoint(String endpoint) {
+        this.endpoint = endpoint;
+        return this;
+    }
+
+    /*
+     * The HTTP pipeline to send requests through
+     */
+    private HttpPipeline pipeline;
```

Comment:
Having your member variables intertwined between methods is hard to reason about what member variables are actually in a class. Usually constants, variables, methods are grouped together and then by visibility.